source/common/json/json_loader.cc

```diff
@@ -646,7 +646,8 @@ bool ObjectHandler::Uint(unsigned value) {
 }
 bool ObjectHandler::Int64(int64_t value) { return handleValueEvent(Field::createValue(value)); }
 bool ObjectHandler::Uint64(uint64_t value) {
-  if (value > std::numeric_limits<int64_t>::max()) {
+  uint64_t max_int = std::numeric_limits<int64_t>::max();
```

Comment:
nit: instead of having a local variable, can you just static cast the result to `uint64_t` ? Same elsewhere?

Reply:
Ok, done!